src/pylorax/ltmpl.py

```diff
@@ -503,9 +503,35 @@ def installpkg(self, *pkgs):
             pkgs = pkgs[1:]
             required = True
 
+        excludes = []
+        while '--except' in pkgs:
+            idx = pkgs.index('--except')
+            if len(pkgs) == idx+1:
+                raise ValueError("installpkg needs an argument after --except")
+
+            excludes.append(pkgs[idx+1])
+            pkgs = pkgs[:idx] + pkgs[idx+2:]
+
```

Comment:
It looks for the --except entries anywhere in the list and pulls them out into the excludes list. So if you wanted you could interleave pkgglobs and --excepts.